src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -1619,6 +1619,16 @@ private void OnFileChanged(FileSystemEventArgs e)
                  string.Compare(fileName, ScriptConstants.ProxyMetadataFileName, StringComparison.OrdinalIgnoreCase) == 0) ||
                 !_directorySnapshot.SequenceEqual(Directory.EnumerateDirectories(ScriptConfig.RootScriptPath)))
             {
+                // CRI ICM: 46695121
```

Comment:
I'm considering an alternative to this now that will achieve your same goal in a generalized, reusable way. Please hold off on this until I try that out